client/src/link-scripts.js

```diff
@@ -0,0 +1,26 @@
+// Get the links and add event listeners
```

Comment:
Out of curiosity, you've used vanilla JavaScript to manage links on the page. While it certainly works, in a React application, we typically handle events and interactions using React's built-in capabilities. Is there a specific reason for choosing this approach over using React's event handling?